### DISCUSSION TITLE
Routing

### DISCUSSION THREAD

**Discussion by jonwalz** [jonwalz/crypto-data#14] [General]
We'll need to decide on what to use for page routing. Here are a couple good options: 

### React location
- https://react-location.tanstack.com/
- This one is a bit newer but has some good feature promises

### Reach router
- https://reach.tech/router/
- This one is well known in the industry and battle tested

**Comment by jonfreihofer** [comment]
I'm more familiar with Reach, so a bit more inclined to go that route (pun intended)

**Reply by jonwalz** [reply]
Let's do it
